docs/inside-arbitrum-nitro/inside-arbitrum-nitro.md

```diff
@@ -364,7 +364,7 @@ Some Arbitrum nodes will choose to act as _validators_. This means that they wat
 
 Not all nodes will choose to do this. Because the rollup protocol doesn’t decide what the chain will do but merely confirms the correct behavior that is fully determined by the inbox messages, a node can ignore the rollup protocol and simply compute for itself the correct behavior. For more on what such nodes might do, see the [Full Nodes](#full-nodes) section.
 
-Being a validator is permissionless--anyone can do it. Offchain Labs provides open source validator software, including a pre-built Docker image.
+Offchain Labs provides open source validator software, including a pre-built Docker image. The underlying protocol supports permissionless validation, i.e.,--anyone can do it. (Currently on Arbitrum One, validators/stakers are whitelisted; see ["mainnet beta status"](../mainnet-beta.md)).
```

Comment:
Is it currently the case that anyone can run a watchtower validator?

Reply:
Watchtower validator does not post anything on-chain, so anyone can run validator in watchtower mode.
The PR https://github.com/OffchainLabs/nitro/pull/1141 adds instructions for simple watchtower mode.  2.0.6 is also being prepared to fix a corner case when running validator in local docker.